### PR TITLE
Fix affiliations

### PR DESCRIPTION
**Pull request by Matthew-Jennings**
Add cities to affiliations
Add "the" to RAH
Expand USA

As per https://github.com/openjournals/joss-reviews/issues/4555#issuecomment-1272261778

-----

Can you all please check and, if necessary, correct the edits I've made to your particular affiliation? Some additional notes:

@SimonBiggs, would you mind confirming whether "CancerCare Partners" is appropriate for @msobolewski, given Riverina Cancer Centre is listed for Jacob? Also, is it "Wagga Wagga" if we use CCP, or their headquarters?

@pchlap, I thought it might be better to expand to the full name for the Ingham Institute, but please feel free to revert to the original wording.

### CODE REVIEW

**Comment by guanfada** [comment]
Mine is correct. Thank you @Matthew-Jennings

**Review by sjswerdloff — APPROVED** [review]
That is the correct city for my affiliation

**Review by ayalalazaro — APPROVED** [review]
Mine is correct too.

**Comment by kingrpaul** [comment]
From:   Painless Skin Cancer Treatment, Mississippi, USA
To: Painless Skin Cancer Treatment Center, Meridian, Mississippi, United States of America

Thanks.

**Review by guanfada — CHANGES_REQUESTED** [review]
add "Connecticut, " after "New Haven, "

**Review by pchlap — APPROVED** [review]
Thanks, @Matthew-Jennings, yes the full name of Ingham is better.

File: paper.md [hunk 1]
```diff
@@ -37,42 +37,42 @@ authors:
...
+  - name: IRCCS Humanitas Research Hospital, Milan, Italy
     index: 12
-  - name: Humanitas University, Italy
+  - name: Humanitas University, Milan, Italy
     index: 13
-  - name: Saskatchewan Cancer Agency, Canada
+  - name: Saskatchewan Cancer Agency, Saskatoon, Canada
     index: 14
-  - name: CancerCare Partners, Australia
+  - name: CancerCare Partners, Wagga Wagga, Australia
```
Comment (SimonBiggs) [inline paper.md:75]:
```suggestion
  - name: CancerCare Partners, Sydney, Australia
```

File: paper.md [hunk 2]
```diff
@@ -37,42 +37,42 @@ authors:
...
+  - name: The Royal Adelaide Hospital, Adelaide, Australia
     index: 2
-  - name: ELEKTA Pty Ltd, New Zealand
+  - name: ELEKTA Pty Ltd, Auckland, New Zealand
     index: 3
-  - name: University of New South Wales, Australia
+  - name: University of New South Wales, Sydney, Australia
     index: 4
-  - name: Ingham Institute, Australia
+  - name: Ingham Institute for Applied Medical Research, Liverpool, Australia
```
Comment (SimonBiggs) [inline paper.md:55]:
```suggestion
  - name: Ingham Institute for Applied Medical Research, Sydney, Australia
```

**Comment by Matthew-Jennings** [comment]
@SimonBiggs, are you sure about the Ingham Institute "city" change? 
image

**Review by crcrewso — APPROVED** [review]

**Comment by SimonBiggs** [comment]
> @SimonBiggs, are you sure about the Ingham Institute "city" change? image

I stand corrected:

https://en.m.wikipedia.org/wiki/City_of_Liverpool_(New_South_Wales)

**Review by guanfada — APPROVED** [review]

**Review by rembishj — DISMISSED** [review]
sorry for the delay with the ORCID... mine is 0000-0002-6508-4175

**Pull request merged**